include/gatekeeper_sol.h

```diff
@@ -143,6 +143,8 @@ struct sol_config {
 	 * Configuration files should not refer to them.
 	 */
 
+	rte_atomic32_t     ref_cnt;
```

Comment:
Need one more space to align with other fields. Also, may add a comment for this field?